guides/aws/cloud-practitioner/README.md

```diff
@@ -0,0 +1,107 @@
+# AWS Certified Cloud Practitioner Playbook
+
+If you are looking to learn about cloud fundamentals and gain an insight into AWS Cloud, you have come to the right place. This guide will help and support you through your journey to becoming an AWS Certified Cloud Practitioner.
+
+The AWS Certified Cloud Practitioner examination is targeted towards individuals who want to demonstrate that they have the foundational knowledge of cloud computing within AWS.
+
+## Guide
+
+You can use the following list to check where you are in terms of studying for the AWS Certified Cloud Practitioner, or what additional knowledge you need in order to schedule an exam. By the end of your studies, you should be able to:
+
+* Define what the AWS Cloud is and the basic global infrastructure
+* Describe basic AWS Cloud architectural principles
+* Describe the AWS Cloud value proposition
+* Describe key services on the AWS platform and their common use cases
+* Describe basic security and compliance aspects of the AWS platform and the shared security model
```

Comment:
I think the term they use is ["Shared Responsibility Model"](https://aws.amazon.com/compliance/shared-responsibility-model/)

Reply:
This was taken directly from AWS so I would leave it as it is. The list outlines abilities that are validated by the certification.

Source: https://aws.amazon.com/certification/certified-cloud-practitioner